src/ompl/tools/benchmark/src/Benchmark.cpp

```diff
@@ -691,6 +691,7 @@ void ompl::tools::Benchmark::benchmark(const Request &req)
                     break;
             }
         }
+        planners_[i]->clear();
```

Comment:
Why the clear here? Aren't the planners cleared at the top of the loop?

Reply:
Yes that is true. But when I was using it to benchmark a lot of planner (like 30+), some planners did not clear their last run, which then filled up my memory. Since I did not had so much memory on my laptop, and I was fearing this would affect the results of planners which were at the very end. This clear() call just makes sure that all the temporary structures are deleted again (and nothing remains from the last run).